### PR TITLE
Modal speed-up

### PR DESCRIPTION
**Pull request by vxsx**
The goal is to speed up the modal js execution and it's perceived performance.
Has some potential breaking changes if users used undocumented features, such as classnames etc.

- [x] nothing is broken
- [x] sourcemaps are not committed

modalDuration must be in sync with css for the moment

### CODE REVIEW

**Comment by vxsx** [comment]
@FinalAngel ready for review

File: cms/static/cms/js/modules/cms.modal.js [hunk 1]
```diff
@@ -42,12 +37,32 @@
...
                     this._events();
                 }
 
                 // ready modal
-                this.modal.data('ready', true);
+                this.ui.modal.data('ready', true);
+            },
+
+            _setupUI: function setupUI() {
+                this.ui = {};
```
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.modal.js]: why don't you just do:

```
this.ui = {
    modal: ...
    body: ...
}
```

?

File: cms/static/cms/sass/components/_loader.scss [hunk 2]
```diff
@@ -2,9 +2,9 @@
 // #LOADER#
 #cms-toolbar {
     .cms-loader {
-        background: $color-window url("../img/loader.gif") no-repeat center center !important;
+        background: #efefef url("../img/loader.gif?1") no-repeat center center !important;
```
Comment (FinalAngel) [inline cms/static/cms/sass/components/_loader.scss]: add random string

File: cms/templates/admin/cms/page/plugin/change_form.html [hunk 3]
```diff
@@ -1,3 +1,13 @@
 {% extends "admin/change_form.html" %}
-{% load cms_admin %}
-{% block submit_buttons_bottom %}{% submit_row_plugin %}{% endblock %}
\ No newline at end of file
+{% load cms_admin admin_static %}
+
+{% block extrastyle %}
+    {{ block.super }}
+    {% if request.GET.modal %}
+        <link rel="stylesheet" type="text/css" href="{% static "cms/css/cms.toolbar.modal.css" %}">
```
Comment (FinalAngel) [inline cms/templates/admin/cms/page/plugin/change_form.html]: single quotes inside

**Comment by yakky** [comment]
@vxsx great one!

**Pull request merged**